src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: $trn-duration;
```

Comment:
Better place it before all nested elements.
Add prop names, that are animated.

Reply:
fixed